test/metabase/query_processor/middleware/add_dimension_projections_test.clj

```diff
@@ -11,19 +11,21 @@
 ;;; ----------------------------------------- add-fk-remaps (pre-processing) -----------------------------------------
 
 (def ^:private example-query
-  {:database (mt/id)
-   :type     :query
-   :query    {:source-table (mt/id :venues)
-              :fields       [[:field-id (mt/id :venues :price)]
-                             [:field-id (mt/id :venues :longitude)]
-                             [:field-id (mt/id :venues :category_id)]]}})
+  (delay
+    {:database (mt/id)
+     :type     :query
+     :query    {:source-table (mt/id :venues)
+                :fields       [[:field-id (mt/id :venues :price)]
+                               [:field-id (mt/id :venues :longitude)]
+                               [:field-id (mt/id :venues :category_id)]]}}))
```

Comment:
should port these too: should not call `mt/id` before tests start running -- is there a way to enforce this?